dags/catalyst.py

```diff
@@ -0,0 +1,46 @@
+"""
+DAG to schedule generation of performance reports for recently completed nimbus experiments.
+
+See the [jetstream repository](https://github.com/mozilla/catalyst).
+
+"""
+
+from datetime import datetime, timedelta
+
+from airflow import DAG
+
+from operators.gcp_container_operator import GKEPodOperator
+from utils.tags import Tag
+
+default_args = {
+    "owner": "dpalmeiro@mozilla.com",
+    "email": [
+        "dpalmeiro@mozilla.com",
+    ],
+    "depends_on_past": False,
+    "start_date": datetime(2025, 3, 18),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 1,
+    "retry_delay": timedelta(minutes=30),
+}
+
+tags = [Tag.ImpactTier.tier_1]
```

Comment:
I think I'd call this tier 2 or 3, but let me see if I can find some sort of documented guidance.

Reply:
Ok, here are the docs: https://github.com/mozilla/bigquery-etl/blob/main/docs/reference/airflow_tags.md

Based on this, I'd suggest Tier 2, but open to your opinion obviously.

Reply:
Tier 2 sounds good here.